app/const.py

```diff
@@ -6,6 +6,11 @@ class MIMETYPE(StrEnum):
     JSON = "application/json"
 
 
+FUND_TYPE_ID_TO_FRIENDLY_NAME = {
+    "TD": "Towns Deal",
```

Comment:
Fix typo. Would be better if this came from the API for better accuracy but this is OK for now.
```suggestion
    "TD": "Town Deal",
```